content/teams/Tstar/index.md

```diff
@@ -0,0 +1,146 @@
+Besides this , other mental health issues tackled by the solution are , anger issues, anxiety , various kinds of phobias, depression , demotivation , stress and to bundle it all up a guided meditation experience.
+Main tools used in this app are , unity3D engine , Google’s VR SDK, Android SDK, Azure, firebase (to be implemented).
+The solution’s USP is there is no such product in the market, its ease of use (just wear a headset), accessibility (works on the cheapest phones) , headset is readily available and is cheap , the app is free.
+There is immense potential for scaling this app, it can be used as a tool by psychiatrists to treat patients, patients can use it to take care of themselves, most features are available offline hence no server side setup is required, there is no such app that does VR mental health care (closest competitions are audio only solutions).
+This app can change the mental health care landscape of india by leveraging the VR tech and making it more accessible, potential is endless.
+
+Demo - https://youtu.be/Ax7dE_bSAsU
+
+More details - https://drive.google.com/file/d/1JiIjQ2llw0X2mn_BYni1FgbUc0H-Ec12/view?usp=sharing
+
+_**Tech-Stack-Used**_ :   Unity Game Engine C#
+
+_**GitHub-Link**_ :   https://github.com/Neelkukreti/CodeCampSubmission.git 
+
+_**Hosted-Link**_ :    https://drive.google.com/file/d/1OmgyR80H_UFbu2APdmX_GBf-2ok84uYt/view?usp=sharing
+
+---
+
+# Member Details
+
+_**No of members**_ : 2
+
+Fill Team Details for only the valid no of members in your team and delete the leftover Template...
+
+## Leader Details
+
+_**Name**_ :Neel Kukreti
+
+_**University-Name**_ : JIIT Noida
+
+_**country**_ : India
+ 
+_**Technical-Skills**_ : Unity, Game dev, VR, AR , web dev , Android ,React JS
+
+_**Specialization**_ :	Game development
+
+_**GitHub-ID**_ :  https://github.com/Neelkukreti
+
+---
+
+## Member 2 Details
+
+_**Name**_ :	Tanvi Thakur
+
+_**University-Name**_ : JIIT Noida
+
+_**country**_ : India
+ 
+_**Technical-Skills**_ : AI, ML , Blockchain , Unity , AR , VR
+
+_**Specialization**_ : ML
+
+_**GitHub-ID**_ :   https://github.com/tanvi070299
+
+---
+
+## Member 3 Details
+
+_**Name**_ :
+
+_**University-Name**_ : 
+
+_**country**_ :
+ 
+_**Technical-Skills**_ :
+
+_**Specialization**_ :
+
+_**GitHub-ID**_ :   
+
+---
+
+## Member 4 Details
+
+_**Name**_ :
+
+_**University-Name**_ : 
+
+_**country**_ :
+ 
+_**Technical-Skills**_ :
+
+_**Specialization**_ :
+
+_**GitHub-ID**_ :  
+
+---
+
+## Member 5 Details
+
+_**Name**_ :
+
+_**University-Name**_ : 
+
+_**country**_ :
+ 
+_**Technical-Skills**_ :
+
+_**Specialization**_ :
+
+_**GitHub-ID**_ :  
+
+---
+
+## Member 6 Details
+
+_**Name**_ :
+
+_**University-Name**_ : 
+
+_**country**_ :
+ 
+_**Technical-Skills**_ :
+
+_**Specialization**_ :
+
+_**GitHub-ID**_ :  
+
+
```

Comment:
Delete Extra left over template

Do these changes and I will merge it later.
remember 10 PM is the deadline for Today's PR session.